doc/guide.rst

```diff
@@ -555,6 +555,12 @@ phased_snvs
     The number of biallelic, heterozygous SNVs that are marked as phased in the input VCF.
     This is a subset of *phased*.
 
+pct_phased
+    The percentage of heterozygous variants that are phased. Same as *phased* / *heterozygous_variants*.
+
```

Comment:
You should distinguish between a fraction and a percentage: The percentage is the fraction times 100. The formula here describes the fraction, but the text says percentage.

I haven’t made up my mind whether it makes more sense to report percentages or fractions in general, so I’ll leave it up to you, but it should match the description and be consistent.

For readability, I would like to not abbreviate percent to pct.

Also, for a different project, I read a standard for a tabular file format (reporting V/D/J rearrangements). They say that their column names follow the convention that the [scope narrows from left to right](https://docs.airr-community.org/en/v1.3.1/datarep/rearrangements.html#custom-columns). I think this is a good way to do it, and we somewhat follow this convention already here, so to be consistent, we should call the new columns `phased_percentage` (or `phased_fraction`) and `phased_snvs_percentage` (or `phased_snvs_fraction`).

Reply:
> You should distinguish between a fraction and a percentage: The percentage is the fraction times 100. The formula here describes the fraction, but the text says percentage.

Yes, this was a bit sloppy by me. 

> I haven’t made up my mind whether it makes more sense to report percentages or fractions in general, so I’ll leave it up to you, but it should match the description and be consistent.

For me reporting fractions in the TSV makes more sense. In the stdout I think it is nicer to show percentage since it is easier to read. 

> Also, for a different project, I read a standard for a tabular file format (reporting V/D/J rearrangements). They say that their column names follow the convention that the [scope narrows from left to right](https://docs.airr-community.org/en/v1.3.1/datarep/rearrangements.html#custom-columns). I think this is a good way to do it, and we somewhat follow this convention already here, so to be consistent, we should call the new columns `phased_percentage` (or `phased_fraction`) and `phased_snvs_percentage` (or `phased_snvs_fraction`).

Thanks for sharing this! I have not thought too deeply about this before but this make more sense indeed.